content/features/proxy2/configure.md

```diff
@@ -0,0 +1,411 @@
+
+The configuration file is a Lua script that extends the standard proxy route library. Advanced users can modify the route library itself, or even replace it entirely with their own Lua code. The techniques described on this page focus on using the route library as-is, which covers the majority of Memcached proxy use-cases.
+
+### Overview of the configuration file
+
+The configuration file has four general parts:
+
+1. Optional invocations of global functions to turn on logging or debugging features.
+1. An optional `settings{}` block, which sets various proxy settings to values other than their defaults.
+1. A `pools{}` block, which defines groups of backend Memcached servers that the proxy routes queries to.
+1. One or more `routes{}` blocks, which lay out the rules by which the proxy routes requests to the pools defined in the `pools{}` blocks.
+
+    It's common to have only one `routes{}` block. If you want to have the router handle requests differently depending upon connection context, [you can define multiple `routes{}` blocks](#multiple-trees).
+
+The following sections detail the contents of each of the above parts, with examples.
+
+### Enable logging and debugging
+
+To enable logging and debugging features of the proxy, include any of the following function calls at the top of the configuration file:
+
+* `verbose(1)` has Memcached print additional information to standard output when loading the proxy configuration. This setting also lets you use the `say({{<var>}}TEXT{{</var>}})` function to print <var>TEXT</var> to standard output while loading your custom configuration.
+
+* `debug(1)` turns on a debugging mode that is mainly useful if you are modifying the routing library. This setting also lets you use the `dsay({{<var>}}TEXT{{</var>}})` function to print <var>TEXT</var> to standard output while loading the proxy configuration.
+
+### Adjust proxy settings
+
+To fine-tune various proxy behaviors, include a `settings{}` block in your configuration file. The block contains any number of function calls that set values on the `mcp` object. For the full list of available setting functions, see [General API documentation]({{<legacy_proxy_base_path>}}#general-api-documentation).
+
+For example, the following settings block sets a ten-second connection timeout, and an upper limit of five active requests:
+
+```lua
+settings {
+    mcp.backend_connect_timeout(10)
+    mcp.active_req_limit(5)
+}
+```
+
+### Define backend pools
+
+Use the required `pools{}` block to define the locations and other attributes of the back-end Memcached servers that the router directs queries to.
+
+The `pools{}` block contains one or more pool definitions in the following format:
+
+```lua
+{{<var>}}POOL_NAME{{</var>}} = {
+    backend_options = { {{<var>}}BACKEND_OPTIONS{{</var>}} },
```

Comment:
there's also an `options` key for pool-specific options, ie:
https://github.com/memcached/memcached-proxylibs/blob/main/lib/routelib/examples/pooloptions.lua

Reply:
Added with https://github.com/memcached/memcached.github.io/pull/3/commits/5698658af884c6fed376fb937b18f0b78a9ae1cb.